### ISSUE TITLE
Support for Heltec CubeCell series

### ISSUE THREAD

**Issue by SebDominguez** [platformio/platformio-core#3078]
I'm requesting [this board](https://heltec.org/project/htcc-ab01/) to be added into PlatformIO.

This board is relatively new and the MCU is low power. It's based on an ASR6501 and unfortunately this platform doesn't exist yet.

Heltec has already made this board [Arduino compatible](https://github.com/HelTecAutomation/ASR650x-Arduino)

**Comment by Skipper-is** [comment]
Would be great for this to be on platformio

**Comment by irukard** [comment]
Indeed! This is a great, low power dev board with LoRa radio.

**Comment by jacobsonjn** [comment]
Also would like this

**Comment by crapougnax** [comment]
Looking forward to!

**Comment by lnlp** [comment]
@ivankravets This request is open for 5 months already, but there appears to be no progress yet. Can you give some information on when CubeCell support will be added?

**Comment by ivankravets** [comment]
@wasn-eu @Heltec-Aaron-Lee could you help with integrating your software & hardware products?

**Comment by Heltec-Aaron-Lee** [comment]
Is there any guy who can share some platformIO migration quick start manual? We will finish in the next two weeks.

**Comment by lnlp** [comment]
Have a look here: https://docs.platformio.org/en/latest/platforms/custom_platform_and_board.html

**Comment by Heltec-Aaron-Lee** [comment]
Thank you

**Comment by ivankravets** [comment]
@Heltec-Aaron-Lee glad to see you here! Quick guide:

1) Create a new repository `https://github.com/HelTecAutomation/platform-asrmicro` in your organization and invite me (thanks!)
2) See examples of open source dev-platforms => https://github.com/topics/platformio-platform
3) Clone `https://github.com/HelTecAutomation/platform-asrmicro` to `~/.platformio/platforms/asrmicro`. You can work on integration here and commit changes to the repository.

This repository is a good starting point https://github.com/platformio/platform-atmelmegaavr
You just need to correct packages in `platform.json` to use ARM toolchain. Check other dev-platforms where we use this toolchain. For example, https://github.com/platformio/platform-atmelsam

Thanks!

**Comment by eni23** [comment]
@Heltec-Aaron-Lee any update on this?

**Comment by jacobsonjn** [comment]
No new updates on this?

**Comment by samy4sam** [comment]
TThe ASR Microelectronics ASR605x platform and some CubeCell boards are available on Platformio since yesterday. But I get an error message when I try to install the ASR605x platform

`PIO Core Call Error: "PlatformManager: Installing asrmicro650x\r\n\n\nError: Detected unknown package 'asrmicro650x'"`

[https://docs.platformio.org/en/latest/platforms/asrmicro650x.html](url)

**Comment by ivankravets** [comment]
We have not released it yet. We are in the process. Could you try the latest development version?

1. Please open PlatformIO IDE Terminal and type:
```
pio platform install https://github.com/HelTecAutomation/platform-asrmicro650x.git
```
2. Use any of these examples https://github.com/HelTecAutomation/platform-asrmicro650x/tree/develop/examples

Does it work? If yes, we will do the first 1.0.0 release. Thanks!

**Comment by ivankravets** [comment]
We've just published 0.1.0 version into the our registry https://github.com/HelTecAutomation/platform-asrmicro650x/releases/tag/v0.1.0

Please try again.
